### PR TITLE
Remove the generated / compiled js files from source control.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by peternewman** [comment]
I don't think we should do this. Similar to ola.js/mobile.js in OLA, it means people don't have to have the tools to fiddle with both the front and back end, if they only want to work on one half.

**Comment by daveol** [comment]
we already require the user to download the libraries with bower (a node.js tool), so I think it's not a problem to remove the minified js files. LGTM

**Comment by daveol** [comment]
a instruction for minification before upload in the readme would be nice

**Comment by peternewman** [comment]
Ah I didn't realise that @daveol although I could manually download those libs (however I see even Debian Experimental doesn't include recent enough libs https://packages.debian.org/experimental/libjs-angularjs . It's only 1.3.13-1 and we want 1.3.15)!

**Comment by peternewman** [comment]
I guess I could write a little tool to grab them from the website instead.

**Comment by peternewman** [comment]
FWIW, I'm now someone who tries to only install packages globally as debs, even if that means making a deb locally for them.

**Comment by nomis52** [comment]
Updated README with more info.

**Comment by nomis52** [comment]
Unfortunately the web world has moved to npm. I think if people are changing the code it's a fair requirement for them to have the necessary tools installed.

File: README.md [hunk 1]
```diff
@@ -4,12 +4,24 @@ This is the codebase for the rdm.openlighting.org website.
...
 
-## Get dependencies
+## Development Setup
 
-Before you upload to appengine or start developing you'll have to download some javascript dependencies.
-These can be downloaded using [nodejs](https://github.com/joyent/node)'s package manager [npm](https://github.com/npm/npm).
-The packages get downloaded using [bower](https://github.com/bower/bower) and [grunt](https://github.com/gruntjs/grunt)
-You have to install grunt-cli globally yourself by running either
+On the production instance, the javascript is minified using
+[UglifyJS](http://lisperator.net/uglifyjs/).  Before you start changing on the
```
Comment (peternewman) [inline README.md]: SPaG double space
SPaG changing the

File: README.md [hunk 2]
```diff
@@ -4,12 +4,24 @@ This is the codebase for the rdm.openlighting.org website.
...
+code, or deploy to appengine you'll need to download the javascript tools that
+are used for this process.
+
+The tools are managed using [nodejs](https://github.com/joyent/node)'s package
+manager [npm](https://github.com/npm/npm).  The packages are downloaded using
+NPM & [bower](https://github.com/bower/bower) and bower is called using
```
Comment (peternewman) [inline README.md]: Is it npm or NPM?

Comment (peternewman) [inline README.md]: SPaG and

Comment (peternewman) [inline README.md]: SPaG s/and bower/; bower/

File: README.md [hunk 3]
```diff
@@ -4,12 +4,24 @@ This is the codebase for the rdm.openlighting.org website.
...
+[grunt](https://github.com/gruntjs/grunt)
+
+If all of this seems rather complex don't despair, one everything is installed
```
Comment (peternewman) [inline README.md]: SPaG once

File: README.md [hunk 4]
```diff
@@ -17,53 +29,89 @@ or
...
+
+The newer Angular code is unit-tested using
+[karma](https://github.com/karma-runner/karma). The tests are in
+unit-test-js/tests/rdm.js and the karma configuration in
+unit-test-js/karma.conf.js. The test can be run with:
 ```bash
 grunt unit-test
 ```
-It does require firefox and the dependencies to be installed to be able to run the test
+It does require firefox and the dependencies to be installed to be able to run
```
Comment (peternewman) [inline README.md]: SPaG Firefox

Comment (peternewman) [inline README.md]: Does it also require a GUI? Mention GUI free options.

File: README.md [hunk 5]
```diff
@@ -17,53 +29,89 @@ or
...
+
+If you want an automatic copy when the files change, use:
 ```bash
 grunt copy-watch
 ```
-### Important!
-When you are done debugging don't forget to run
+
+#### Important!
+When you are finished debugging to run
```
Comment (peternewman) [inline README.md]: SPaG remember to, or remove the to

File: README.md (hunk 1 above)
Comment (nomis52) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (nomis52) [inline README.md]: Done

File: README.md (hunk 3 above)
Comment (nomis52) [inline README.md]: Done

File: README.md (hunk 4 above)
Comment (nomis52) [inline README.md]: I believe it does. @daveol  ?

File: README.md (hunk 5 above)
Comment (nomis52) [inline README.md]: Done

File: README.md (hunk 4 above)
Comment (daveol) [inline README.md]: yes it does, a GUI free option is [phantomjs](http://phantomjs.org/) it's karma runner is [here](https://github.com/karma-runner/karma-phantomjs-launcher)

**Pull request merged**